worker.py

```diff
@@ -494,188 +496,304 @@ def __user_menu(self):
                 self.__help_menu()
 
     def __order_menu(self):
-        """User menu to order products from the shop."""
-        log.debug("Displaying __order_menu")
-        # Get the products list from the db
-        products = self.session.query(db.Product).filter_by(deleted=False).all()
         # Create a dict to be used as 'cart'
         # The key is the message id of the product list
         cart: Dict[List[db.Product, int]] = {}
-        # Initialize the products list
-        for product in products:
-            # If the product is not for sale, don't display it
-            if product.price is None:
-                continue
-            # Send the message without the keyboard to get the message id
-            message = product.send_as_message(w=self, chat_id=self.chat.id)
-            # Add the product to the cart
-            cart[message['result']['message_id']] = [product, 0]
-            # Create the inline keyboard to add the product to the cart
-            inline_keyboard = telegram.InlineKeyboardMarkup(
-                [[telegram.InlineKeyboardButton(self.loc.get("menu_add_to_cart"), callback_data="cart_add")]]
-            )
-            # Edit the sent message and add the inline keyboard
-            if product.image is None:
-                self.bot.edit_message_text(chat_id=self.chat.id,
-                                           message_id=message['result']['message_id'],
-                                           text=product.text(w=self),
-                                           reply_markup=inline_keyboard)
-            else:
-                self.bot.edit_message_caption(chat_id=self.chat.id,
-                                              message_id=message['result']['message_id'],
-                                              caption=product.text(w=self),
-                                              reply_markup=inline_keyboard)
-        # Create the keyboard with the cancel button
-        inline_keyboard = telegram.InlineKeyboardMarkup([[telegram.InlineKeyboardButton(self.loc.get("menu_cancel"),
-                                                                                        callback_data="cart_cancel")]])
-        # Send a message containing the button to cancel or pay
-        final_msg = self.bot.send_message(self.chat.id,
-                                          self.loc.get("conversation_cart_actions"),
-                                          reply_markup=inline_keyboard)
-        # Wait for user input
+        CategoryMode = self.cfg["Mode"]["category_mode"]
+        FinalStep = False
```

Comment:
Variable names should be `snake_case` (all lowercase, with underscores as spaces)

```suggestion
        category_mode = self.cfg["Mode"]["category_mode"]
        final_step = False
```